<!--
DESCRITPION
-->


<launch>
        <node pkg="ira_laser_tools" name="laserscan_multi_merger" type="laserscan_multi_merger" output="screen">
		<param name="destination_frame" value="/cart_frame"/>
		<param name="cloud_destination_topic" value="/merged_cloud"/>
		<param name="scan_destination_topic" value="/scan_multi"/>
                <param name="laserscan_topics" value ="scandx scansx" /> <!-- LIST OF THE LASER SCAN TOPICS TO SUBSCRIBE -->
	</node>
</launch>
